com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs

```diff
@@ -74,7 +74,7 @@ protected override bool DoShadersStripper(HDRenderPipelineAsset hdrpAsset, Shade
             // we allow user to make explicit request for it and it bypass other request
             if (!hdrpAsset.currentPlatformRenderPipelineSettings.supportRuntimeAOVAPI)
```

Comment:
just for awareness. When we use our AOV API at runtime, we force this settings. This mean that we still have a behavior per configuration for HDRP. It is ok like this